dist/svg.js

```diff
@@ -1540,10 +1542,27 @@ SVG.FX = SVG.invent({
       return this
     }
 
+    // set the internal animation pointer at the start position and updates the visualisation
+  , atStart: function() {
+    // Cast boolean to number
+    return this.at(+this.situation.reversed)
+  }
+
+    // set the internal animation pointer at the end position and updates the visualisation
+  , atEnd: function() {
+    // Cast boolean to number
+    return this.at(+!this.situation.reversed)
+  }
+
     // set the internal animation pointer to the specified position and updates the visualisation
   , at: function(pos){
       var durDivSpd = this.situation.duration/this._speed
 
+      // correct position
+      if(pos > 1) pos = 1
+      if(pos < 0) pos = 0
+      if(this.situation.reversed) pos = 1 - pos
```

Comment:
Well something like this but hopefully better (I'm on my smartphone so it is hard for me to code XD):

```javascript
 , step: function(ignoreTime){

      // convert current time to position
      if(!ignoreTime) this.pos = this.timeToPos(+new Date)
      else {
        var durDivSpd = this.situation.duration/this._speed

        // correct position
        if(pos > 1) pos = 1
        if(pos < 0) pos = 0
        if(this.situation.reversed) pos = 1 - pos

        this.situation.start = +new Date - pos * durDivSpd
        this.situation.finish = this.situation.start + durDivSpd
      }

      if(this.pos >= 1 && (this.situation.loop === true || (typeof this.situation.loop == 'number' && --this.situation.loop))){

        if(this.situation.reversing){
          this.situation.reversed = !this.situation.reversed
        }

        return this.at(this.situation.reversed ? 1-(this.pos-1) : (this.pos-1))
      }

      if(this.situation.reversed) this.pos = 1 - this.pos

      // correct position
      if(this.pos > 1)this.pos = 1
      if(this.pos < 0)this.pos = 0
```